packages/cli/src/commands/certs/command.ts

```diff
@@ -0,0 +1,116 @@
+import { Command } from '../help';
+import { packageName } from '../../util/pkg-name';
+
+export const certsCommand: Command = {
+  name: 'certs',
+  description:
+    'Interact with SSL certificates. This command is intended for advanced use only. By default, Vercel manages your certificates automatically.',
+  arguments: [
+    {
+      name: 'command',
+      required: false,
+    },
+  ],
+  subcommands: [
+    {
+      name: 'ls',
+      description: 'Show all available certificates',
+      arguments: [],
+      options: [],
+      examples: [],
+    },
+    {
+      name: 'issue',
+      description: ' Issue a new certificate for a domain',
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** 

```suggestion
      description: 'Issue a new certificate for a domain',
```